.gitignore

```diff
@@ -49,3 +49,5 @@ ehthumbs_vista.db
 
 # Windows shortcuts
 *.lnk
+package-lock.json
```

Comment:
package-lock.json should not be ignored, otherwise you don't get deterministic builds. Not a huge deal for this project, but still. I'm gonna revert this change.